### PR TITLE
Form validation: Fixed form validation i18n build

### LINKED ISSUE #6924: Forms: Example shows validation message only in English
Slight problem with the current example page for form validation.  The [French page](http://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-fr.html) is showing part of the validation message in English (see image attached).

errorformvalidationlanguage

### PR DESCRIPTION
**Pull request by pjackson28**
Fixes #6924

### CODE REVIEW

**Comment by pjackson28** [comment]
@LaurentGoderre This didn't seem to fix the issue (thought it had). Could you please take a look and see why it isn't appending the jQuery validation error messages any more?

**Comment by pjackson28** [comment]
Could it be related to both minified and unminified files being in the same directory?

**Comment by pjackson28** [comment]
Hold on, I think I found some paths that needed to be changed as well.

**Comment by pjackson28** [comment]
Nope, that didn't fix it either. Any ideas?

**Comment by LaurentGoderre** [comment]
I will take a look

**Comment by pjackson28** [comment]
Figured it out and pushed up a fix. Turned out to be a compare mismatch due to the paths used.

File: Gruntfile.coffee [hunk 1]
```diff
@@ -386,7 +386,7 @@ module.exports = (grunt) ->
 			i18n:
 				options:
 					process: ( src, filepath ) ->
-						lang = filepath.replace "dist/js/i18n/", ""
+						lang = filepath.replace "dist/wet-boew/js/i18n/", ""
```
Comment (LaurentGoderre) [inline Gruntfile.coffee]: It wasn't working with `coreDist`?

Comment (pjackson28) [inline Gruntfile.coffee]: Unfortunately no. It doesn't resolve <code>&lt;%=coreDist %&gt;</code>, it just leaves that as a string.

Comment (LaurentGoderre) [inline Gruntfile.coffee]: That's weird. It should....

Comment (LaurentGoderre) [inline Gruntfile.coffee]: AAAh I know!!! `lang = filepath.replace grunt.config("coreDist") + "js/i18n/", ""`

Comment (pjackson28) [inline Gruntfile.coffee]: Yes that seems to work nicely. I updated the PR.

**Pull request merged**